Added clarifications for "Start from beginning" and "Start from timestamp" options (#108540)
Azure/azure-content: articles/cosmos-db/analytical-store-change-data-capture.md

- There's no limitation around the fixed data retention period for which changes are available

> [!IMPORTANT]

> > Please note that if "Start from beginning" option is selected, you will get an initial load of full snapshot of container data in the first run, followed by capturing changed / incremental data in the subsequent runs. Similar is the case when "Start from timestamp" option is selected - the initial load will process the data from the given timestmp, followed by capturing incremental / changed data in the subsequent runs. The `Capture intermediate updates`, `Capture Deletes` and `Capture Transactional store TTLs` under [source options](get-started-change-data-capture.md) tab determine if intermediate updates and deletes are captured in Sinks.

Reviewer: ```suggestion
> Please note that if the "Start from beginning" option is selected, the initial load includes a full snapshot of container data in the first run, and changed or incremental data is captured in subsequent runs. Similarly, when the "Start from timestamp" option is selected, the initial load processes the data from the given timestamp, and incremental or changed data is captured in subsequent runs. The `Capture intermediate updates`, `Capture Deletes` and `Capture Transactional store TTLs`, which are found under the [source options](get-started-change-data-capture.md) tab, determine if intermediate updates and deletes are captured in sinks.
```
edit for grammar and clarity
